packages/plugin-docker/index.js

```diff
@@ -3,23 +3,43 @@ import {
   DOCKER_ENTRYPOINT,
 } from '@octolinker/helper-grammar-regex-collection';
 import relativeFile from '@octolinker/resolver-relative-file';
+import resolverTrustedUrl from '@octolinker/resolver-trusted-url';
 
-export function dockerHubUrl(target) {
+function isURL(str) {
+  // RegExp origin https://stackoverflow.com/a/3809435/2121324
+  const expression =
+    /(https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|www\.[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9]\.[^\s]{2,}|https?:\/\/(?:www\.|(?!www))[a-zA-Z0-9]+\.[^\s]{2,}|www\.[a-zA-Z0-9]+\.[^\s]{2,})/gi;
+  const urlRegex = new RegExp(expression);
+
+  return urlRegex.test(str);
+}
```

Comment:
I copied this function from [packages/helper-normalise-resolver-results](https://github.com/OctoLinker/OctoLinker/blob/4b722a9784b7d0571412c960052623cb77fc4248/packages/helper-normalise-resolver-results/index.js#L5). We might want to consider to create a shared utils package, but since it's only used in two places it might be fine. Happy to discuss this further.